src/WorkBC.Web/ClientApp/projects/jb-search/src/app/job-detail/job-detail.component.html

```diff
@@ -191,6 +191,7 @@ <h3>{{skill.Category.Name}}</h3>
             </div>
             <!-- How To Apply -->
             <form>
+              <br />
```

Comment:
A more sustainable way to address styling issues is via CSS, as opposed to incidental HTML tags. No big deal for this trivial fix, but please keep that in mind for future styling fixes.